lib/comm_qmp.cpp

```diff
@@ -301,10 +220,16 @@ int comm_query(MsgHandle *mh)
   return (QMP_is_complete(mh->handle) == QMP_TRUE);
 }
 
+template <typename T>
```

Comment:
this is not used with QMP or do I miss that somewhere? Maybe an incomplete commit?

Reply:
Yeah, this looks like an incomplete commit, the `std::accumulate`s haven't been replaced further in the file.

Reply:
Fixed